components/application-operator/charts/application/templates/tests/application-gateway-tests.yaml

```diff
@@ -1,27 +1,30 @@
-apiVersion: v1
-kind: Pod
+{{- if .Capabilities.APIVersions.Has "testing.kyma-project.io/v1alpha1" }}
```

Comment:
This Helm test is triggered by Application Operator Tests. I assume it will not be run after this change, which is not desired.